android/src/facebook/TiFacebookModule.java

```diff
@@ -278,7 +278,20 @@ public void onCompleted(GraphResponse response) {
 		request.setHttpMethod(method);
 		request.executeAsync();
 	}
-	
+    
+	@Kroll.method
+    	public void setPushNotificationsRegistrationId(String token) {
+		AppEventsLogger.setPushNotificationsRegistrationId(token);
+	}
+
+	@Kroll.method
+	public void logPushNotificationOpen(KrollDict parameters, String action) {
+		AppEventsLogger logger = AppEventsLogger.newLogger(TiApplication.getInstance().getCurrentActivity());
+		Bundle paramBundle = Utils.mapToBundle(parameters);
+		
+		logger.logPushNotificationOpen(paramBundle,action);
```

Comment:
make `String action` optional, check against `null` and call `logPushNotificationOpen(paramBundle)` if it's null, the existing method otherwise.